README.md

```diff
@@ -52,13 +56,13 @@ for the user-handled portions of SAML profiles. See "Metadata" and "Plugins" for
 * If testing an IdP:
   * Provide your IdP's metadata file path to the `samlconf` script using `-i` or `--idpMetadata`.
   * Configure your IdP with the test kit's SP metadata from
-  `<PATH_TO_PROJECT>/distribution/command-line/src/main/resources/test-sp-metadata.xml`
+  `distribution/command-line/src/main/resources/test-sp-metadata.xml`
```

Comment:
✏️ Should really reference the `distribution/command-line/build/install/samlconf/conf` location (assuming `buildDist` was run, or just the `samlconf/conf` directory if an archive was exploded.